### PR TITLE
Add basic Ubuntu CI

### PR DESCRIPTION
**Pull request by zpervan**
Add a basic CI workflow for Linux Ubuntu.

### CODE REVIEW

File: .github/workflows/ubuntu_ci.yml
Comment (Seviel) [inline .github/workflows/ubuntu_ci.yml:34]: Could you please add new line at the end of the file unless it would cause incorrectness.

**Comment by Seviel** [comment]
Hi Zvonimir, is there a way to test this PR before merging?

**Comment by zpervan** [comment]
Sadly, no. You need to merge it in order for the CI to start building. It's a bit tedious to set up the CI in the beginning, but once done - no need to adjust it anymore.

**Comment by zpervan** [comment]
Also, one thing to consider here is to have a standardized way of naming branches, i.e. `feature/my-awesome-features` or `bugfix/stupid-bug` so the CI can be triggered on a pull request. 

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request

**Comment by zpervan** [comment]
> Hi Zvonimir, is there a way to test this PR before merging?

Sadly, no. Github Actions explicitly says that it needs approval:
image

**Review by Seviel — APPROVED** [review]
Ok, we can give it a shot, thanks for the contribution.

**Comment by zpervan** [comment]
Thanks for the approval, but sadly I don't have write access to your project so I cannot merge it. You need to do it.

**Pull request merged**
